src/scripts/store.ts

```diff
@@ -23,39 +30,69 @@ export default class Store {
         });
     }
 
+    getDeletedUsers(): number[] {
```

Comment:
One minor suggestion. 

```suggestion
    getDeletedUsers(): Array<number> {
```

Reply:
Fixed it.
https://github.com/dhruvdhaduk-simform/user-management-ts/pull/5/commits/ee7c8b7ba67d5ec005f2a3dc03b4ec922196176e